dev/products/nidem/nidem.yaml

```
@@ -0,0 +1,18 @@
+name: nidem
+description: National Intertidal Digital Elevation Model 25m 1.0.0
+metadata_type: eo
+
+metadata:
+    format:
+        name: GeoTIFF
+    product_type: item_v2.0
+measurements:
+-   dtype: float32
+    name: 'nidem'
+    nodata: -9999
+    units: '1'
```

```suggestion
    units: 'metres'
```
NIDEM is an elevation in